sap_hana/tests/test_integration.py

```diff
@@ -42,18 +44,19 @@ def test_check_invalid_schema(aggregator, instance, dd_run_check):
         any_order=False,
     )
 
-    assert check.log.error.call_count == 9
-
     for call_args in check.log.error.call_args_list:
         assert "invalid schema name: UNKNOWN_SCHEMA" in call_args[0][2]
 
+    assert check.log.error.call_count == 9
```

Comment:
moving this assertion to the end because the other one gives more information in cause of failure